Backend/Backend/Models/User.cs

```diff
@@ -103,5 +103,5 @@ public string GetGenderString()
             _ => "None",
         };
     }
-    
+  
```

Comment:
Try not to put files you didnt change in the pr. Not a huge deal here, but its bad practice

Reply:
![image](https://github.com/awaazo/awaazo/assets/78050750/3237f27b-b329-49b9-841e-8866796fa9d4)
this kinda demonstrates what I mean lol